### PR TITLE
Add powered by coffee story

### PR DESCRIPTION
**Pull request by mooreds**

### CODE REVIEW

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+* The FusionAuth REST API
+* FusionAuth's OAuth support
+* Social Login
+* Migration and importing of user data
+
+**Dan:** What problems did we solve for you? And how were you solving them before FusionAuth?
+
+**Stewart:** FusionAuth solved a problem of scale and access for us. WordPress is a great CMS but it's a terrible user manager.
+
+By using FusionAuth to handle User authentication and access control we can bypass the WordPress User system and handle Authentication entirely client side. That means we can be much more aggressive with our caching, improving overall performance.
```
Comment (dpolstra) [inline site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:46]: entirely on the client side.

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+* Social Login
+* Migration and importing of user data
+
+**Dan:** What problems did we solve for you? And how were you solving them before FusionAuth?
+
+**Stewart:** FusionAuth solved a problem of scale and access for us. WordPress is a great CMS but it's a terrible user manager.
+
+By using FusionAuth to handle User authentication and access control we can bypass the WordPress User system and handle Authentication entirely client side. That means we can be much more aggressive with our caching, improving overall performance. 
+
+Similarly, these users don't need to have logins to WordPress anymore, hugely improving overall performance as we take a user table down from hundreds of thousands of users, down to a few hundred authors and editors.
```
Comment (dpolstra) [inline site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:48]: table down from hundreds of thousands of users, ~down~ to a few hundred

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+> I think the cost savings purely from helping optimise caching and performance on high traffic WordPress sites, massively reducing the number of servers needed to run a performant site, is massive.
+
+**Dan:** Why did you choose FusionAuth over the competition?
+
+**Stewart:** In our case we were introduced to Fusion Auth by a client who had already decided to use it.
+
+We've continued to use and recommend Fusion Auth because we find it very pleasant to use as developers.  There are lots of options for API access, different ways we can integrate FA in our projects and support is always there when we need a little extra help.
+
+It's great to see the product continue to evolve and the road map be published. It gives us more options to support our own clients.
```
Comment (dpolstra) [inline site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:58]: roadmap

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+**Stewart:** Impossible to say I'm afraid. I think the cost savings purely from helping optimise caching and performance on high traffic WordPress sites, massively reducing the number of servers needed to run a performant site, is massive.
+
+Second, the reduction in developer time, having simple, straightforward API's to share that user profile and access information in multiple accessory applications is beyond calculation.
+
+> WordPress is a great CMS but it's a terrible user manager.
+
+**Dan:** How do you run FusionAuth (k8s, standalone tomcat server, behind a proxy, etc)?
+
+**Stewart:** We run a couple of different Fusion Auth setups; we have a managed Fusion Auth instance that Fusion Auth lookafter for production work and we also have FusionAuth running as a Docker Container in our Development Network. We've also run FusionAuth as a service in a Docker Compose driven project.
```
Comment (dpolstra) [inline site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:70]: FusionAuth

Comment (dpolstra) [inline site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:70]: instance that FusionAuth looks after

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+**Dan:** What problems did we solve for you? And how were you solving them before FusionAuth?
+
+**Stewart:** FusionAuth solved a problem of scale and access for us. WordPress is a great CMS but it's a terrible user manager.
+
+By using FusionAuth to handle User authentication and access control we can bypass the WordPress User system and handle Authentication entirely client side. That means we can be much more aggressive with our caching, improving overall performance. 
+
+Similarly, these users don't need to have logins to WordPress anymore, hugely improving overall performance as we take a user table down from hundreds of thousands of users, down to a few hundred authors and editors.
+
+> I think the cost savings purely from helping optimise caching and performance on high traffic WordPress sites, massively reducing the number of servers needed to run a performant site, is massive.
```
Comment (jobannon) [inline site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:50]: Using massively twice sounds strange

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+**Dan:** Why did you choose FusionAuth over the competition?
+
+**Stewart:** In our case we were introduced to Fusion Auth by a client who had already decided to use it.
```
Comment (jobannon) [inline site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:54]: Fusion Auth => FusionAuth

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+We've continued to use and recommend Fusion Auth because we find it very pleasant to use as developers.  There are lots of options for API access, different ways we can integrate FA in our projects and support is always there when we need a little extra help.
```
Comment (jobannon) [inline site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:56]: Fusion Auth => FusionAuth

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+
+It's great to see the product continue to evolve and the road map be published. It gives us more options to support our own clients.
+
+**Dan:** How much time and money would you say FusionAuth has saved you and your clients?
+
+**Stewart:** Impossible to say I'm afraid. I think the cost savings purely from helping optimise caching and performance on high traffic WordPress sites, massively reducing the number of servers needed to run a performant site, is massive.
```
Comment (jobannon) [inline site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:62]: massive twice

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+
+Second, the reduction in developer time, having simple, straightforward API's to share that user profile and access information in multiple accessory applications is beyond calculation.
```
Comment (jobannon) [inline site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:64]: Second, the reduction in developer time and having simple, straightforward API's to share user profile and access information in multiple accessory applications is beyond calculation.

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md (hunk 4 above)
Comment (jobannon) [reply site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:70]: we have a managed Fusion Auth instance that Fusion Auth lookafter for production work

I am confused by this sentence fragment

**Review by jobannon — APPROVED** [review]
Interesting Piece!

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md (hunk 4 above)
Comment (mooreds) [reply site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:70]: I think he's talking about FusionAuth cloud.

File: site/_posts/2021-06-29-powered-by-coffee-fusionauth.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+**Stewart:** We use Fusion Auth to:
+
+* Centralise user data and identity so profiles can be used across products and services.
+* Manage access to content on membership sites.
+* Provide all the user infrastructure for a site, out of the box.
+* Give users a central place to manage their preferences.
+* Control access to the WordPress admin panel and setting user access levels.
+
+To do this, we used these features:
+* The FusionAuth REST API
```
[the remark this answers is not in the document]
Comment (mooreds) [reply site/_posts/2021-06-29-powered-by-coffee-fusionauth.md:37]: I think `REST API` is ok too.

**Pull request merged**